### DISCUSSION TITLE
problem in mapping in android!

### DISCUSSION THREAD

**Discussion by Keimelpinto2550** [ppy/osu#13569] [Q&A] [answered]
i can't mapping in android cuz i can't put the background img and the audio, how can i do this?

**Comment by Game4all** [comment]
Please see  #11219.  Unfortunately, mobile support ~for editor~ in general isn't a big priority so this will have to wait unless someone decides to implement it.

**Comment by bdach** [comment, marked as answer]
Did you try clicking on the "Background" and "Audio Track" text boxes? They should pull up a built-in file selector dialog.

**Reply by Keimelpinto2550** [reply]
current directory: COMPUTER :)
